src/connection/connection_sr.c

```diff
@@ -1157,38 +1163,36 @@ css_connect_to_master_server (int master_port_id, const char *server_name, int n
 #else /* WINDOWS */
       /* send the "pathname" for the datagram */
       /* be sure to open the datagram first.  */
+      pname = std::filesystem::temp_directory_path ();
+      pname += "/cubrid_tcp_setup_server" + std::to_string (getpid ());
```

Comment:
Why didn't you use the newly created mechanism?

Reply:
Come to think of it, it can't be used. If filesys_temp.hpp has a function that can do this, it seems more consistent.

Reply:
there is a task opened to refactor the code from connection_cl and connection_sr in the connect functions as is already 80% identical.